ramalama

```diff
@@ -215,6 +215,7 @@ def list_cli(ramalama_store, args):
 
 
 funcDict["list"] = list_cli
+funcDict["ls"] = list_cli
```

Comment:
Ollama has ls as an alias to list, so trying to keep with "norms" from other tools